packages/backend/src/services/ProjectService/ProjectService.test.ts

```diff
@@ -144,10 +144,10 @@ describe('ProjectService', () => {
             expect(formatValue('percent', undefined, 0.05)).toEqual('5%');
             expect(formatValue('percent', undefined, '5')).toEqual('500%');
             expect(formatValue('percent', undefined, 'foo')).toEqual('foo');
-            expect(formatValue('percent', undefined, false)).toEqual(false);
+            expect(formatValue('percent', undefined, false)).toEqual('false');
```

Comment:
I think this might break `boolean` displays on tables https://github.com/lightdash/lightdash/blob/main/packages/frontend/src/utils/tableData.ts

Reply:
Tables are using the formattedValue returned by the BE which is now formatting all values based on field type.